examples/src/main/r/data-manipulation.R

```diff
@@ -75,8 +75,8 @@ destDF <- select(flightsDF, "dest", "cancelled")
 
 # Using SQL to select columns of data
 # First, register the flights SparkDataFrame as a table
-registerTempTable(flightsDF, "flightsTable")
-destDF <- sql(sqlContext, "SELECT dest, cancelled FROM flightsTable")
+createOrReplaceTempView(flightsDF, "flightsTable")
```

Comment:
Hmm I assumed this was a breaking change in 2.0 in Spark SQL for Scala and Python as well.  But looking at https://github.com/apache/spark/pull/12945 it looks like the call is only deprecated and its not a breaking change - In that case we should add it back with a similar deprecation warning

cc @liancheng